src/runtime/pkg/oci/utils.go

```diff
@@ -1046,3 +1083,58 @@ func (a *annotationConfiguration) setUintWithCheck(f func(uint64) error) error {
 	}
 	return nil
 }
+
+// CalculateSandboxSizing will calculate the number of CPUs and amount of Memory that should
+// be added to the VM if sandbox annotations are provided with this sizing details
+func CalculateSandboxSizing(spec *specs.Spec) (numCPU, memSizeMB uint32) {
+	if spec == nil || spec.Annotations == nil {
+		return 0, 0
+	}
+
+	// if the annotation isn't defined, or if there's an error in parsing, we'll get a result of zero,
+	// resulting in the numCPU and/or memory being set to zero (ie, noop), so let's just continue silently
```

Comment:
Hmm. Please, give me some log about this when this happens.  We don't need to fail, but info log, which would only be displayed when we enable debug, would be enough to help debugging possible issues with this.

Reply:
This is actually the "normal" scenario, where the annotation is not provided. I think a debug log for what the retrieved values are, however (so you can see we are setting as 0 versus a set value?)

Reply:
Actually, let me just expand each parse in order to check and log a warning. I was excited to be concise, but that isn't necessarily best here!

Reply:
Thanks, @egernst.